### PR TITLE
1946403B additional updates to Upgrade Guide 

### PR DESCRIPTION
**Pull request by emarcusRH**
Fixes issue BZ#1946403

Changes proposed in this pull request:

- added step to updating Manager - update Manager subscriptions to 4.4 (RHV Docs only)
- improve formatting of upgrade options table in Overview chapter
- move up the recommendation for the RHV Upgrade Helper, format as a NOTE (RHV Docs only)

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by: (please @dcdacosta )

### CODE REVIEW

File: source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc [hunk 1]
```diff
@@ -158,6 +158,21 @@ If the backup contained grants for extra database users, this command creates th
...
+ifdef::4-3_local_db,4-3_remote_db,4-3_SHE[]
+link:{URL_customer-portal}{URL_docs}{URL_lang-locale}{URL_product_rhv}4.3/html-single/installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_command line/index#Enabling_the_Red_Hat_Virtualization_Manager_Repositories_install_RHVM[Enabling the {virt-product-fullname} {engine-name} Repositories] for {virt-product-fullname} 4.3.
+endif::4-3_local_db,4-3_remote_db,4-3_SHE[]
+ifdef::SHE_minor_updates,migrating_to_SHE,minor_updates[]
+link:{URL_virt_product_docs}{URL_format}installing_{URL_product_virt}_as_a_standalone_manager_with_local_databases/index#Enabling_the_Red_Hat_Virtualization_Manager_Repositories_install_RHVM[Enabling the {virt-product-fullname} {engine-name} Repositories] for {virt-product-fullname} 4.4.
+endif::SHE_minor_updates,migrating_to_SHE,minor_updates[]
++
+Updates to the {virt-product-fullname} {engine-name} are released through the Content Delivery Network.
+endif::rhv-doc[]
+
```
Comment (stoobie) [inline source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc:175]: @emarcusRH  I don't understand why you're discussing enabling 4.2 repos in the 4.4 RHVM upgrade topic.

**Review by stoobie — CHANGES_REQUESTED** [review]
One comment.

File: source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc (hunk 1 above)
Comment (emarcusRH) [reply source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc:175]: > @emarcusRH I don't understand why you're discussing enabling 4.2 repos in the 4.4 RHVM upgrade topic.

my mistake - I pulled in too many links while copying another entry...

**Review by stoobie — APPROVED** [review]
LGTM

**Pull request merged**